V2/time_tracker/time_entries/_application/_time_entries/_get_time_entries_summary.py

```diff
@@ -0,0 +1,60 @@
+import json
+from http import HTTPStatus
+import datetime
+
+import azure.functions as func
+
+from ... import _domain
+from ... import _infrastructure
+from time_tracker._infrastructure import DB
+from time_tracker.utils.enums import ResponseEnums
+
+
+def get_time_entries_summary(req: func.HttpRequest) -> func.HttpResponse:
+    database = DB()
+    time_entry_dao = _infrastructure.TimeEntriesSQLDao(database)
+    time_entry_service = _domain.TimeEntryService(time_entry_dao)
+    use_case = _domain._use_cases.GetTimeEntriesSummaryUseCase(time_entry_service)
+
+    response_params = {
+        "body": ResponseEnums.NOT_FOUND.value,
+        "status_code": HTTPStatus.NOT_FOUND,
```

Comment:
should go HTTPStatus.NOT_FOUND.value if not take everything as 200, at same for others status

Reply:
This is not corrected yet